build_runner/lib/src/package_graph/package_graph.dart

```diff
@@ -167,6 +171,7 @@ PackageDependencyType _dependencyType(source) {
       case 'hosted':
         return PackageDependencyType.hosted;
       case 'path':
+      case 'sdk': // Until Flutter supports another type, assume same as path.
```

Comment:
I have no context here, is it actually equivalent to a path dependency? Or is it coming from pub?